functions/Restore-DbaDatabase.ps1

```diff
@@ -299,7 +354,7 @@ function Restore-DbaDatabase {
 						# We need to use Test-DbaSqlPath and other commands instead
 						# Prevent people from trying 
 						
-						#Write-Warning "Currently, you can only use UNC paths when running this command remotely. We expect to support non-UNC paths for remote servers shortly."
+						#Stop-Function -silent:$silent -message "Currently, you can only use UNC paths when running this command remotely. We expect to support non-UNC paths for remote servers shortly."
```

Comment:
-Silent:$silent is not needed - it is inherited from the command's primary params. Also, please cap -Message.